api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java

```diff
@@ -53,23 +53,26 @@
 
     private Extension ext5;
 
-
     @Before
     public void setUp() throws Exception {
 
+    	// clear up from previous tests
```

Comment:
@Wyclif, DirtiesContext in openmrs 2.0 causes the database to be emptied. Tests are not able to proceed because authentication is not possible. I agree we need DirtiesContext to maintain the tests in appframework but we need to make changes to the platform to support DirtiesContext. Particularly, I proposed a change where if Authentication in core failed, the database was reloaded.
I think a more senior dev has to solve the problem with DirtiesContext before this ticket can be resolved.
It is possible that DirtiesContext is not a problem, but this really needs to be verified by a senior dev because I am seeing a problem.